salt-mode.el

```diff
@@ -82,6 +82,50 @@ suitable for spellchecking."
 
 (put 'salt-mode 'flyspell-mode-predicate #'salt--flyspell-predicate)
 
+(defun salt--state-module-at-point ()
```

Comment:
The name of the package is `salt-mode`, so in theory every symbol should start with `salt-mode-`. Unfortunately we already have `salt-indent-level` and `salt--flyspell-predicate` (sorry), and the `salt` custom group. Maybe it's best to fix all of those now before it's too late? For `salt-indent-level` it's a backwards-incompatible change, but a minor one.